### ISSUE TITLE
Let user hide menubar like vscode

### ISSUE THREAD

**Issue by nguyenkien** [beekeeper-studio/beekeeper-studio#495]
I would like Beekeeper hide menubar, and only show when I press Alt key (Similar to what vscode do). That would save some space, as it doesn't provide much useful thing. 
I don't want to use client side render, because that would also remove window shadow. And I don't like window controls (close, maximize) look differently with other app.

**Additional context**
This is what Vscode does
Normally look like this
![Screenshot from 2020-12-27 23-28-09](https://user-images.githubusercontent.com/1940081/103175280-58178800-489b-11eb-95e9-ae2355498f22.png)
When I press Alt key the menu show
![Screenshot from 2020-12-27 23-28-30](https://user-images.githubusercontent.com/1940081/103175286-61085980-489b-11eb-9fd0-41395ece92dc.png)

**Comment by rathboma** [comment]
Agreed we should support this. Should just be a case of hooking up the right electron functions.

So, we don't support that yes, but you can go to view -> menu style and change it to `client`. That will give you some fancy menus that live in the titlebar instead.

Is that good enough?

**Comment by nguyenkien** [comment]
There are problems with client side decorate, doesn't have drop shadow on xfce and window control (close, minimize button) not look like other apps.

**Comment by rathboma** [comment]
You should be able to download an AppImage from https://github.com/beekeeper-studio/beekeeper-studio/actions/runs/466890918 when this PR build is done. Can you grab it and let me know if it works right for you?

**Comment by nguyenkien** [comment]
Thank for work in this issue. But how do I download it?
